service/src/main/java/uk/gov/hmcts/reform/fpl/service/hearing/ManageHearingsOthersGenerator.java

```diff
@@ -0,0 +1,66 @@
+package uk.gov.hmcts.reform.fpl.service.hearing;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.reform.fpl.model.CaseData;
+import uk.gov.hmcts.reform.fpl.model.HearingBooking;
+import uk.gov.hmcts.reform.fpl.model.Other;
+import uk.gov.hmcts.reform.fpl.model.common.Element;
+import uk.gov.hmcts.reform.fpl.model.order.selector.Selector;
+import uk.gov.hmcts.reform.fpl.service.FeatureToggleService;
+import uk.gov.hmcts.reform.fpl.service.OthersService;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import static uk.gov.hmcts.reform.fpl.enums.YesNo.NO;
+import static uk.gov.hmcts.reform.fpl.enums.YesNo.YES;
+import static uk.gov.hmcts.reform.fpl.utils.ElementUtils.unwrapElements;
+
+@Component
+@RequiredArgsConstructor(onConstructor_ = {@Autowired})
+public class ManageHearingsOthersGenerator {
+    private final OthersService othersService;
+    private final FeatureToggleService toggleService;
+
+    public Map<String, Object> generate(CaseData caseData, HearingBooking hearingBooking) {
+
+        List<Element<Other>> allOthers = caseData.getAllOthers();
+        List<Element<Other>> selectedOthers = hearingBooking.getOthers();
+
+        Map<String, Object> data = new HashMap<>();
+
+        if (toggleService.isServeOrdersAndDocsToOthersEnabled()) {
+            if (!allOthers.isEmpty()) {
+                data.put("hasOthers", YES.getValue());
+                data.put("othersSelector", buildOtherSelector(allOthers, selectedOthers));
+                data.put("others_label", othersService.getOthersLabel(caseData.getAllOthers()));
+                data.put("sendOrderToAllOthers",
+                    sendOrderToAllOthers(allOthers, selectedOthers) ? YES.getValue() : NO.getValue());
+            }
```

Comment:
you should just be able to use the Yes/No value here from "sendOrderToAllOthers" in case data..

Reply:
Not sure that would be the best idea, what if they create multiple hearings with different others selected? 

If they create one hearing where all others are selected and then create a new hearing, the value would just be pre-populated based on the previous hearing which would not be relevant

Reply:
Yes i forgot your ticket needs it prepopulated, so in this case, it works better as is